src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -148,32 +149,50 @@ struct PlantHydraulicsParameters{
     conductivity_model::CP
     "Water retention model and parameters"
     retention_model::RP
-    "Root distribution function P(z)"
-    root_distribution::F
+    "Rooting depth parameter (m) - a characteristic depth below which 1/e of the root mass lies"
+    rooting_depth::RD
 end
 
 function PlantHydraulicsParameters(;
     ai_parameterization::PrescribedSiteAreaIndex{FT},
     ν::FT,
     S_s::FT,
-    root_distribution::Function,
+    rooting_depth::Union{Nothing, FT, ClimaCore.Fields.Field} = nothing,
+    root_distribution::Union{Nothing, Function} = nothing,
     conductivity_model,
     retention_model,
 ) where {FT}
-    return PlantHydraulicsParameters{
-        FT,
-        typeof(ai_parameterization),
-        typeof(conductivity_model),
-        typeof(retention_model),
-        typeof(root_distribution),
-    }(
-        ai_parameterization,
-        ν,
-        S_s,
-        conductivity_model,
-        retention_model,
-        root_distribution,
-    )
+    if root_distribution isa Nothing
+        return PlantHydraulicsParameters{
+            FT,
+            typeof(ai_parameterization),
+            typeof(conductivity_model),
+            typeof(retention_model),
+            typeof(rooting_depth),
+        }(
+            ai_parameterization,
+            ν,
+            S_s,
+            conductivity_model,
+            retention_model,
+            rooting_depth,
+        )
+    else
+        return PlantHydraulicsParameters{
+            FT,
+            typeof(ai_parameterization),
+            typeof(conductivity_model),
+            typeof(retention_model),
+            FT,
+        }(
+            ai_parameterization,
+            ν,
+            S_s,
+            conductivity_model,
+            retention_model,
+            FT(1.0 / root_distribution(0.0)),
```

Comment:
Not to stir the pot, I know there was just a discussion about breaking changes, but I dont think we should support passing in the root_distribution function. Or, if we do, we need to know for that distribution how to compute the rooting_depth according to our fixed, single, rooting distribution in src (as of this PR). We cant assume that the rooting distribution they provide has the same functional form, from which we know how to compute the depth